db/measurement_units.json

```diff
@@ -613,7 +648,7 @@
     "multiplier": null,
     "coerced_measurement_unit_code": null,
     "original_unit": null,
-    "measurement_unit_type": "volume"
+    "measurement_unit_type": "number"
```

Comment:
Are we sure about this?

"cubic metre" sounds like a volume :)

Reply:
:facepalm: